src/services/AdvancedTradingView/useAdvancedTradingView.hook.ts

```diff
@@ -166,6 +155,13 @@ export const useAdvancedTradingView = () => {
     }
 
     fetchTrades()
+
+    const interval = setInterval(() => {
+      fetchTrades()
+    }, 15000)
```

Comment:
I see we are using `15000` milliseconds all over our app. Don't know if it's better to have a global constant or just a constant for Advanced Trading View.

Reply:
Done, created `REFETCH_DATA_INTERVAL` constant.